lib/utils/scrollTo.js

```diff
@@ -1,14 +1,26 @@
+
+import { detectBrowser } from './index.js';
```

Comment:
See Below, this is kind of an antipattern to what this util does. This util receives all the information int needs from the parent components that consume it. Which is why detect browser is being used higher up the chain. We should be passing the element (body, or a horizontal div) and the method (scrollTop, scrollLeft) down to this function.